### PR TITLE
Add periodic variable examples

### PR DESCRIPTION
**Pull request by mcoughlin**
Start to a periodic variable PR.

### CODE REVIEW

**Review by dmitryduev — COMMENTED** [review]
Thanks for taking the charge! 
Gotta add `field_guide__periodic.md`...

**Comment by mcoughlin** [comment]
@dmitryduev Fail!

File: doc/field_guide__periodic.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+### ZTF light curves
+![ZTF periodic](data/periodic.png)
+
+#### Description
+Periodic variables have periods ranging from minutes to over a thousand days with millimagnitude to magnitude level variations.
+
+#### Light curve characteristics
+- periodic variable
+- period range: 0.001-1000 days
+- amplitude: varies from survey precision to more than a magnitude
```
Comment (dmitryduev) [inline doc/field_guide__periodic.md:23]: Wording? Reminds me of an old Russian joke: new recruits should dig a trench from here to lunchtime.

**Review by dmitryduev — CHANGES_REQUESTED** [review]

File: doc/field_guide__periodic.md (hunk 1 above)
Comment (mcoughlin) [reply doc/field_guide__periodic.md:23]: alright see what you think, i added a definition afterwords.

File: doc/field_guide.md [hunk 2]
```diff
@@ -29,3 +29,6 @@ for more details on the taxonomy.
 
 ```{include} ./field_guide__lpv.md
 ```
+
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply doc/field_guide.md]: ok it is now at the top. i will be sure to move variable from the other PR above this once this is merged.

File: doc/field_guide__periodic.md (hunk 1 above)
Comment (dmitryduev) [reply doc/field_guide__periodic.md:23]: Oops, that was fine, I meant line 23 re amplitude

Comment (dmitryduev) [reply doc/field_guide__periodic.md:23]: I guess it's fine :)

**Review by dmitryduev — APPROVED** [review]

File: config.defaults.yaml [hunk 3]
```diff
@@ -102,6 +102,10 @@ docs:
       coordinates: [9.1523, 68.0221]
       period: 212.0066035
       title: A Mira (Long Period Variable)
+    periodic:
+      coordinates: [288.5243027, 1.51191]
+      period: 0.175036406641672
```
Comment (przemekmroz) [inline config.defaults.yaml:107]: The period is ~2 times longer: 0.350075 days

**Review by przemekmroz — CHANGES_REQUESTED** [review]

**Pull request merged**